lib/install/kubelet/kubelet.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016-2018 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubelet
+
+import (
+	"context"
+	"io/ioutil"
+
+	"github.com/vmware/vic/lib/config"
+)
+
+func ReadKubeletConfigFile(ctx context.Context, configSpec *config.VirtualContainerHostConfigSpec) error {
+	pathName := configSpec.KubeletConfigFile
+	data, err := ioutil.ReadFile(pathName)
+
```

Comment:
whitespace.  It's a go-ism to do the error check immediately after the call without a whitespace line.